configure.ac

```diff
@@ -174,7 +174,7 @@ case "${host_os}" in
 esac
 AC_SUBST(LIBPOSTFIX)
 
-
+CFLAGS="$CFLAGS -std=c99"
```

Comment:
I would add a comment here, otherwise this important setting may get lost in the lengthy `configure.ac` file
```
# 
# set C language mode to C99 
#
```

What about adding also `-pedantic-errors` ? 
(Not sure if it is a good idea, but maybe this can be used for more consistent behaviour)

Reply:
I think that is best left to another series, where we also enable -Wall -Wextra

Reply:
You are right: `-pedantic-error`, as other warning/error CFLAGS do not belong in `configure.ac` but are better specified on the `./configure` invocation.

Reply:
And BTW, I didn't notice how many GNU extensions are already present in the codebase, so `-pedantic-error` introduces to many errors.